components/JournalCard.js

```diff
@@ -91,9 +117,56 @@ const DeleteButton = styled.button`
 
   &:hover {
     color: var(--color-form-foreground);
+  }
+`;
+
+const StyledDialogOverlay = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: rgba(0, 0, 0, 0.5);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const StyledDialogBox = styled.div`
+  background: var(--color-background);
+  padding: 20px;
+  border-radius: 8px;
+  text-align: center;
+  box-shadow: 0 0 10px var(--color-shadow);
+  body.dark-theme & {
+    color: var(--color-foreground);
+  }
+`;
 
-    body.dark-theme & {
-      color: var(--color-highlighted-foreground);
-    }
+const StyledButtonConfirm = styled.button`
+  margin: 10px;
+  padding: 5px 10px;
+  background-color: #e74c3c;
+  color: white;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+
+  &:hover {
+    background-color: #c0392b;
+  }
+`;
+
+const StyledButtonCancel = styled.button`
+  margin: 10px;
+  padding: 5px 10px;
+  background-color: #95a5a6;
+  color: white;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+
+  &:hover {
+    background-color: #7f8c8d;
   }
```

Comment:
Instead of having two buttons that have 90% identical styling, you could have created only one button component that changes the background-color based on a prop. Like this:

```suggestion
const StyledButton = styled.button`
  margin: 10px;
  padding: 5px 10px;
  background-color: ${({ $variant }) => ($variant === "cancel" ? "#95a5a6" : "#e74c3c")};
  color: white;
  border: none;
  border-radius: 5px;
  cursor: pointer;

  &:hover {
    background-color: ${({ $variant }) => ($variant === "cancel" ? "#7f8c8d" : "#c0392b")};
  }
`;
```

In your JSX you would then call this `StyledButton` and pass it either `${variant}="cancel"` or `${variant}="confirm"` as a prop to achieve the same result as before, but now with a reusable component that you can extend, if you decide to have more than 2 variants.